src/components/AutocompleteTypeField.jsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import Autocomplete from "react-autocomplete";
-import "../css/Autocomplete.css";
+import "../CSS/Autocomplete.css";
 
```

Comment:
This is wrong and needs to be moved back to css/ and notCSS

Reply:
@sgrg1 your local folder needs to be named.

Reply:
done